assets/src/styles/vendors/photoswipe/photoswipe.scss

```diff
@@ -0,0 +1,207 @@
+/*! PhotoSwipe main CSS by Dmitry Semenov | photoswipe.com | MIT license */
```

Comment:
This file is based on a file the photoswipe lib provides as a starting point, if I'm correct? Does it have any modifications compared to that file?

 Maybe you can link to the file this is based on? Ideally if they host it in Github we can link to the commit you used.